test/travis_script.sh

```diff
@@ -31,23 +31,39 @@ mysql -u swefreq -h 127.0.0.1 -P 3366 swefreq_test < test/data/load_dummy_data.s
 
 
 echo ">>> Test 3. Check that the backend starts"
-cd backend
 
+cd backend
 ../test/01_daemon_starts.sh
+cd ..
 
 
 echo ">>> Test 4. the backend API"
-python route.py --develop 1>http_log.txt 2>&1 &
+coverage run backend/route.py --port=4000 --develop 1>http_log.txt 2>&1 &
 BACKEND_PID=$!
 
+sleep 2 # Lets wait a little bit so the server has started
+
 function exit_handler() {
+    rv=$?
+    # Ignore errors in the exit handler
+    set +e 
     kill -9 $BACKEND_PID
```

Comment:
Does it work with a plain `kill`? `kill -9` is not nice, but it may not matter much here.